NEWS.md

```diff
@@ -43,6 +43,9 @@
 <summary>Developer Notes</summary>
 
 - Adopted `data-raw/data` R Package Convention (#2427)
+- `compute_bsa(height, weight, method` - now uses improved version of DuBois-DuBois forumula for bsa (#2532)  
```

Comment:
So not burying it in the developer notes?  I think telling users might be a better idea now that I am thinking about it

Reply:
I would keep it in the developer notes because the change doesn't affect the users.

I agree with Jeff that we should make it clear that the results are still the same. Otherwise, it may unsettle our users.

What about the following?
```suggestion
- `compute_bsa()` now uses the more common (but equivalent) version of the DuBois-DuBois formula for BSA. The results have not changed. (#2532)  
```